Public/Src/IDE/VsCode/client/README.md

```diff
@@ -17,14 +12,8 @@ In addition to standard features expected from a modern code editor like Syntax
 - Change All Occurrences (Ctrl+F2)
 - Format Document (Alt+Shift+F)
 
-## What's New
-
-- Improvements to the performance of Find All References.
-
-## Version and Updates
-
-- To find out which version of the DScript extension is installed under Visual Studio Code, look at the installed extensions (using Ctrl+Shift+X) and you should find the extension version number next to the extension name.
-- Starting with this release, the DScript extension automatically checks for updates and presents you with a notification allowing you to upgrade to the latest version.
+We would like to extend this plugin with support for target langauges like C++, C# and others. 
+We have a prototype that uses vscode-cpptools extension for C++ But Omnisharp does not yet have a bridge back to vscode. We'll likely hold off until [Build Server Protocol](https://www.bing.com/search?q=build+server+protocol&PC=U531&cvid=20173cc8036d42be9f67f90d11269e49&FORM=ANAB01&setLang=en-US) is a bit more established and hook into that.

```

Comment:
Did you really want to include a bing link here instead of a concrete search result?

Reply:
nope good catch.